components/table/table-col-sort-button.js

```diff
@@ -53,8 +52,8 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				font-size: inherit;
 				letter-spacing: inherit;
 				line-height: 0.9rem;
-				margin: var(--d2l-table-cell-col-sort-button-size-offset, 4px);
-				margin-inline-end: var(--d2l-table-col-sort-button-margin-inline-end);
+				margin-block: 0 var(--d2l-table-cell-col-sort-button-size-offset, 4px);
+				margin-inline: 0 var(--d2l-table-cell-col-sort-button-size-offset, 4px);
```

Comment:
The cell handles adding padding on the top and left, and the component adds margins to the end and bottom in order to handle wrapping.